src/handler.rs

```diff
@@ -50,7 +50,7 @@ fn keystore_handler (keystore: &mut LedgerKeyStore, request: KeyStoreRequest) ->
     match request {
         KeyStoreRequest::ListAccount => {
             let ledger_ids = keystore.discovered_devices();
```

Comment:
This comes back empty if `refresh_dir()` fails for any reason. This would mean that a user could not see all connected ledgers if there is a json error. This is because of the way that we currently manager various ledgers/connectivity, and should be fixed with #15

Reply:
Sure, but this PR still does some good things I think?

Reply:
I think this has been fixed by the new with-ledger changes, and discovered_devices() no longer interacts with json, and should have no reason to fail if the json read is an error